aot-maven-plugin/src/main/java/com/dylibso/chicory/experimental/maven/aot/AotGenMojo.java

```diff
@@ -69,74 +68,78 @@ public class AotGenMojo extends AbstractMojo {
 
     @Override
     public void execute() throws MojoExecutionException {
-        var module = Parser.parse(wasmFile);
-        var result = AotCompiler.compileModule(module, name);
-        var split = name.split("\\.");
+        try {
```

Comment:
Can we change the API without changing the try/catch structure? That will make the diff easier to review.

Reply:
done :-)